.buildkite/run.sh

```diff
@@ -29,7 +29,7 @@ else
 fi
 
 echo "--- Building proxy"
-(cd app && yarn proxy:build)
+(cd app && ELECTRON_ENABLE_LOGGING=1 yarn proxy:build)
```

Comment:
This is intended to stay?

Reply:
Yeah, we have a case of a flaky test, this will allow us to catch it. It's also good to keep it in on CI permanently, so we see all the Electron console messages that would otherwise be hidden on CI.

Reply:
@rudolfs Can we open an issue for that test with some output?

Reply:
@xla https://linear.app/issue/MON-22